engine/pom.xml

```diff
@@ -313,10 +313,11 @@
       ${camunda.artifact}.engine.*;${camunda.osgi.version};-noimport:=true
     </camunda.osgi.export.pkg>
     <camunda.osgi.import.defaults>
-      org.camunda.bpm.engine.variable.*;${camunda.osgi.import.camunda.commons.version},
+      org.camunda.bpm.engine.variable.*,
```

Comment:
One thing that strikes me here: is it still necessary to include this import explicitly at all? It will be covered by the general `*` import, won't it?

Reply:
Yes, you're right. I will remove it